incuna-sass/legacy/_box-sizing.sass

```diff
@@ -1,14 +1,14 @@
-%border-box
+%legacy-border-box
```

Comment:
i guess so that you can differentiate between them and the modern ones when/if you ever need to use it?
avoids name clash as well. I think